src/Microsoft.Extensions.Localization/Internal/AssemblyResourceStringProvider.cs

```diff
@@ -65,6 +63,7 @@ private IList<string> ThrowOrNull(CultureInfo culture, bool throwOnMissing)
         public IList<string> GetAllResourceStrings(CultureInfo culture, bool throwOnMissing)
         {
             var cacheKey = GetResourceCacheKey(culture);
+
```

Comment:
There's no preferred guidelines in this area beyond sometimes-cited preferences such as including a blank line after curly braces and before `return` statements. But in existing code, it's much easier on your reviewer(s) to ignore the presence or absence of blank lines in code you don't have to change. Looks like unnecessary churn otherwise.